packages/onramp-kit/src/types.ts

```diff
@@ -1,14 +1,23 @@
-import { StripePack } from './packs/stripe/StripePack'
+import Safe from '@safe-global/protocol-kit'
+
 import {
+  StripePack,
   StripeSession,
   StripeEvent,
   StripeEventListener,
   StripeOpenOptions
-} from './packs/stripe/types'
+} from './packs/stripe'
+import {
+  MoneriumPack,
+  SafeMoneriumClient,
+  MoneriumEvent,
+  MoneriumEventListener,
+  MoneriumOpenOptions
+} from './packs/monerium'
 
 // The new packs must implement this interface
 export interface SafeOnRampPack<TPack> {
-  init(): Promise<void>
+  init(safeSdk?: Safe): Promise<void>
```

Comment:
This is optional for Stripe kit, but mandatory for Monerium. I would type it like you did in the `open` function below. Use an `options` object that will have different type depending on the provided pack.

Reply:
done